adx/resource_adx_materialized_view.go

```diff
@@ -164,6 +174,12 @@ func resourceADXMaterializedViewCreateUpdate(ctx context.Context, d *schema.Reso
 	if folder, ok := d.GetOk("folder"); ok {
 		withParams = append(withParams, fmt.Sprintf("folder='%s'", folder))
 	}
+	if maxSourceRecordsForSingleIngest, ok := d.GetOk("max_source_records_for_single_ingest"); ok {
```

Comment:
Since the `with` clause here is only valid when initially creating a view (not updating), please add the `&& new` predicate to this similar to `backfill`